### DISCUSSION TITLE
Prisma Errors and Missing Tables After Running LiteLLM -- Possible Re-migration Issue?

### DISCUSSION THREAD

**Discussion by octadion** [BerriAI/litellm#5555] [Q&A]
Hi everyone,

I’m currently working on a FastAPI application that uses Prisma for database management. The application runs fine, and Prisma connects without any issues. However, when I run the following command to start LiteLLM:

```
litellm --config config.yaml --debug
```
I encounter an error related to Prisma, specifically:
```
prisma.errors.RawQueryError: relation "LiteLLM_VerificationToken" does not exist
```
This error seems to be related to Prisma trying to access or drop certain tables. After encountering this error, I noticed that some tables in my database were actually deleted, which was unexpected. I didn’t perform any manual database migrations, and all I did was run the LiteLLM command above.

My questions are:

1. Is it possible that running litellm --config config.yaml --debug somehow triggers a migration process that affects my Prisma tables?
2. Could LiteLLM be modifying the database schema in some way, and if so, how do I prevent this from happening?
3. Has anyone else experienced Prisma tables being dropped after running LiteLLM, and is there a recommended way to restore or re-migrate my database schema safely?

Any help or insights would be greatly appreciated!

Thank you!

**Comment by thedevd** [comment]
Same here, I also encountered this. Actually for me none of the tables are there in database
